app/lib/features/home/widgets/space_chip.dart

```diff
@@ -1,108 +1,91 @@
-import 'package:acter/common/providers/space_providers.dart';
+import 'package:acter/common/providers/room_providers.dart';
 import 'package:acter/router/utils.dart';
 import 'package:acter_avatar/acter_avatar.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
-import 'package:logging/logging.dart';
 import 'package:skeletonizer/skeletonizer.dart';
 
-final _log = Logger('a3::home::space_chip');
-
 class SpaceChip extends ConsumerWidget {
-  final SpaceItem? space;
-  final String? spaceId;
+  final String spaceId;
   final bool onTapOpenSpaceDetail;
   final bool useCompatView;
   final VoidCallback? onTapSelectSpace;
 
   const SpaceChip({
     super.key,
-    this.space,
-    this.spaceId,
+    required this.spaceId,
     this.onTapOpenSpaceDetail = true,
     this.useCompatView = false,
     this.onTapSelectSpace,
   });
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    if (space == null) {
-      if (spaceId == null) {
-        throw L10n.of(context).spaceOrSpaceIdMustBeProvided;
-      }
-      final spaceLoader = ref.watch(briefSpaceItemProvider(spaceId!));
-      return spaceLoader.when(
-        data: (space) => renderSpace(context, space),
-        error: (e, s) {
-          _log.severe('Failed to load brief of space', e, s);
-          return Chip(
-            label: Text(L10n.of(context).loadingFailed(e)),
-          );
-        },
-        loading: () => renderLoading(spaceId!),
-      );
+    if (useCompatView) {
+      return renderCompactView(context, ref);
     }
-    return renderSpace(context, space!);
+    return renderFullChip(context, ref);
   }
 
-  Widget renderLoading(String spaceId) {
+  static Widget loading() {
```

Comment:
Again any reason for this static widget?

Reply:
So it can be reused from the outside without creating an instance